### PR TITLE
Switch to IrodsPath, rename function

### PR DESCRIPTION
**Pull request by chStaiger**
Rename the listing function for tickets
Switch to IrodsPath

### CODE REVIEW

File: ibridges/tickets.py [hunk 1]
```diff
@@ -143,11 +144,11 @@ def delete_ticket(self, ticket: Union[str, irods.ticket.Ticket], check: bool = F
             ticket = self.get_ticket(ticket)
         if ticket.string in self.all_ticket_strings:
             ticket.delete()
-            self.update_tickets()
+            self.list_tickets()
         elif check:
             raise KeyError(f"Cannot delete ticket: ticket '{ticket}' does not exist (anymore).")
 
-    def update_tickets(self) -> list[TicketData]:
+    def list_tickets(self) -> list[TicketData]:
```
Comment (qubixes) [inline ibridges/tickets.py:151]: Are you sure you want to rename it? Now it looks kind of strange to simply call the method without using its contents.

**Review by qubixes — CHANGES_REQUESTED** [review]
Small suggestions, otherwise thanks for picking this one up!

File: ibridges/tickets.py (hunk 1 above)
Comment (chStaiger) [reply ibridges/tickets.py:151]: I found `update_tickets` confusing. We are not updating them but pull the information again from the iRODS server. So it is more a `list` or `refresh`.

Comment (qubixes) [reply ibridges/tickets.py:151]: Okay, then let's go with refresh_tickets if you agree?

**Pull request merged**